dcp_prototype/backend/browser/api/app.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+
+import chalice
+from chalice import Chalice
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "chalicelib"))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from browser.rds.browser_orm import (DBSessionMaker,
+                                     Project, File,
+                                     LibraryPrepProtocol, Tissue, Species,
+                                     LibraryPrepProtocolJoinProject, TissueJoinProject,
+                                     SpeciesJoinProject)
+
+app = Chalice(app_name='browser-api')
+
+
+@app.route('/')
+def index():
+    return "render documentation here"
+
+
+@app.route('/projects')
+def get_projects():
+    session = DBSessionMaker().session()
+    projects = []
+    for project in session.query(Project).all():
+        projects.append({
+            'id': project.id,
+            'title': project.title,
+            'assays': _get_project_assays(session, project.id),
+            'tissues': _get_project_tissues(session, project.id),
+            'species': _get_project_species(session, project.id),
```

Comment:
_get_project_assays, _get_project_tissues,  _get_project_species seem like they would belong in a utils module rather than this api module